src/components/Layout/DashboardWrapper.tsx

```diff
@@ -22,28 +22,35 @@ export const DashboardWrapper = ({ children }: { children: ReactNode }) => {
   const pathAdmin = useFirstPathElement();
   const { data } = useListSites();
 
-  const isHideSideBar = pathname === "/admin" || pathname === "/admin/add";
+  const isPathAdmin = pathname === "/admin"
+  const isPathAdminAdd = pathname === "/admin/add"
+  const isHideSideBar = isPathAdmin || isPathAdminAdd;
+  const getNip46 = localStorage.getItem("__nostrlogin_nip46")
```

Comment:
We can't rely on a private local-store variable of nostr-login. Please use userPubkey variable or addOnAuth function to check if user is authed.